src/main/resources/extension.yml

```diff
@@ -0,0 +1,3 @@
+nodename: extension
```

Comment:
Do you want this to be part of the main codebase? 
Ideally this should be test resources.

Reply:
I have added it as a part of main codebase. I'm still not sure if this should be a part of resources/main code base. WDYT?